os/arch/arm/src/amebasmart/Make.defs

```diff
@@ -270,7 +278,8 @@ endif
 # Wifi
 ifeq ($(CONFIG_AMEBASMART_WIFI),y)
 ifeq ($(CONFIG_ARCH_FPU),y)
-#EXTRA_LIBS += $(TOPDIR)/board/
+EXTRA_LIBS += $(TOPDIR)/board/rtl8730e/src/libs/lib_wpa_lite.a
+EXTRA_LIBS += $(TOPDIR)/board/rtl8730e/src/libs/lib_wifi_common.a
```

Comment:
hi Mr. Chang, I'm not sure whether I misunderstand you. These two libraries would possibly be updated in the future if there's some fixes or changes inside the libraries. If we add them in the .gitignore, seems like git add will skip them.

Reply:
The main .gitignore file which is located at TizenRT root excludes .a files. Because it's usually built.
So, when we upload .a file like this case, we should add it in .gitingore which is located at .a file not to exclude.
Please refer tp1x case.
https://github.com/Samsung/TizenRT/blob/master/os/board/rtl8721csm/src/libs/.gitignore

Reply:
Thank you Mr. Chang, I added the .gitignore file already